CodeEdit/Breadcrumbs/BreadcrumbsView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  BreadcrumbsView.swift
+//  CodeEdit
+//
+//  Created by Lukas Pistrol on 17.03.22.
+//
+
+import SwiftUI
+import WorkspaceClient
+
+struct BreadcrumbsView: View {
+
+	@ObservedObject var workspace: WorkspaceDocument
+	var file: WorkspaceClient.FileItem
```

Comment:
I think the `file` could just be a `let`?

Reply:
true